virtual_rainforest/models/plants/functional_types.py

```diff
@@ -0,0 +1,52 @@
+"""Initial definition of plant functional type classes.
+
+These are likely to become part of pyrealm.
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+from virtual_rainforest.core.config import ConfigurationError
+from virtual_rainforest.core.logger import LOGGER
+
+
+@dataclass(frozen=True)
+class PlantFunctionalType:
+    """Data class containing plant functional type definitions."""
+
+    pft_name: str
+    maxh: float
+
+
+class PlantFunctionalTypes(dict):
+    """Dictionary of plant functional types."""
```

Comment:
I think the two classes `PlantFunctionalType` and `PlantFunctionalTypes` could very easily be mixed up, could you find a more specific name?

Reply:
As discussed, replaced `PlantFunctionalTypes` with `Flora` at least for now.